02-peer-protocol.md

```diff
@@ -1247,10 +1283,43 @@ The accepting node:
   - MAY respond with a `funding_satoshis` value of zero.
   - If it requires the opening node to only use confirmed inputs:
     - MUST set `require_confirmed_inputs`
+  - if the `option_will_fund` tlv was sent in `open_channel2`:
+    - if they decline to contribute funds or to be paid for their
+      contributions:
+      - SHOULD omit the `will_fund` tlv
+    - if they decide to accept the offer:
+      - MUST include a `will_fund` tlv
+      - MUST set `funding_satoshis` to a value greater than 0msat
+      - MAY send `funding_satoshis` less than `requested_sats`
+      - MUST set `lease_fee_base_sat` to the base fee
+        (in satoshi) it will charge for the `funding_satoshis`
+      - MUST set `lease_fee_basis` to the amount
+        (in thousandths of satoshi) it will charge per `funding_satoshi`
```

Comment:
I thought this was basis points, thus ten thousandths of satoshi? Same for `channel_fee_basis_max` below. The formula used in the examples below uses 1/10_000.